.github/workflows/test.yaml

```diff
@@ -0,0 +1,175 @@
+                empty_strings+=$(sed '$a\\' website/locale/nl/LC_MESSAGES/django.po | tac | sed '/^$/N;/\nmsgstr ""$/,/^msgid/!d' | tac)
+                if [[ $empty_strings ]]; then echo $empty_strings && exit 1; else echo "No untranslated strings found."; fi
+
+                # Check for fuzzy translations in .po files
+                grep --include="*.po" --files-with-matches --recursive "#, fuzzy" website && exit 1 || echo "No fuzzy translations found."
+
+    tests:
+        name: Django Tests
+        runs-on: ubuntu-18.04
+        container: thalia/concrexit-dependencies
+        services:
+            postgres:
+                image: postgres:latest
+                env:
+                    POSTGRES_DB: thalia
+                    POSTGRES_USER: postgres
+                    POSTGRES_PASSWORD: postgres
+        steps:
+            - name: Checkout repository
+              uses: actions/checkout@v2
+
+            - name: Install any missing dependencies
+              run: poetry install --no-interaction
+
+            - name: Run internal Django tests
+              run: poetry run python website/manage.py check
+
+            - name: Test the templates
+              run: poetry run python website/manage.py templatecheck --project-only
+
+            - name: Make migrations
+              run: poetry run python website/manage.py makemigrations --no-input --check --dry-run
+
+            - name: Run coverage
+              run: poetry run coverage run website/manage.py test website/
+
+            - name: Report coverage
+              run: |
+                coverage report --fail-under=100 --omit "website/registrations/urls.py" website/registrations/**.py
+                coverage report --fail-under=100 --omit "website/payments/urls.py" website/payments/**.py
+                coverage report
+            
+            - name: Create coverage report
+              run: coverage html --directory=covhtml --title="Coverage Report"
+
+            - name: Save coverage report
+              uses: actions/upload-artifact@v1
+              with:
+                  name: coverage-html
+                  path: covhtml
+
+
+    documentation-tests:
+        name: Documentation Tests
+        runs-on: ubuntu-18.04
+        container: thalia/concrexit-dependencies
+        steps:
+            - name: Checkout repository
+              uses: actions/checkout@v2
+
+            - name: Install any missing dependencies
+              run: poetry install --no-interaction --extras "docs"
+
+            - name: Run doctest
+              working-directory: docs
+              run: poetry run make doctest
+
+            - name: Run Sphinx
+              working-directory: docs
+              run: poetry run sphinx-build -W . _build -j $(nproc)
+
+            - name: Run API docs script
+              working-directory: docs
+              run: poetry run ./generate-apidocs.sh
```

Comment:
This is missing the `git diff --exit-code`. There should be no changes for this job to succeed. Now there is no test, you're just generating the docs.

Reply:
I think that also answers the question from https://github.com/thaliawww/concrexit/pull/2#issuecomment-594867975